x-pack/platform/plugins/shared/fleet/server/routes/cloud_connector/index.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import type { FleetAuthzRouter } from '../../services/security';
+
+import { API_VERSIONS, CLOUD_CONNECTOR_API_ROUTES } from '../../../common/constants';
+import { FLEET_API_PRIVILEGES } from '../../constants/api_privileges';
+
+import { genericErrorResponse } from '../schema/errors';
+
+import { createCloudConnectorHandler, getCloudConnectorsHandler } from './handlers';
+
+export const registerRoutes = (router: FleetAuthzRouter) => {
+  // POST /api/fleet/cloud_connectors
+  router.versioned
+    .post({
+      path: CLOUD_CONNECTOR_API_ROUTES.CREATE_PATTERN,
+      security: {
+        authz: {
+          requiredPrivileges: [
+            {
+              anyRequired: [
```

Comment:
is this the right set of privileges? what kind of users do we expect to be able to manage cloud connectors? (related question: when we have an UI for managing connectors, where will it live?)